### DISCUSSION TITLE
Email reports

### DISCUSSION THREAD

**Discussion by dannyvankooten** [ibericode/koko-analytics#199] [Ideas]
Periodic email reports. 

- Daily, weekly or monthly. At a scheduled time.
- Customizable email template would be nice to have.

**Comment by ghost** [comment]
Is there possibly a shortcode or function to call the dashboard so I can embed it into an email task myself? I looked in the code a bit but didn’t see anything.

**Reply by dannyvankooten** [reply]
Unfortunately that isn't possible, as email clients do not support JS and only support a very weird subset of HTML and CSS. You can use the `KokoAnalytics\Stats` class to query statistics, but would then have to write your own text/HTML version of the email.

**Comment by dannyvankooten** [comment]
![image](https://github.com/ibericode/koko-analytics/assets/885856/8f953491-8a4c-419d-9c03-6001ba71685c)
![image](https://github.com/ibericode/koko-analytics/assets/885856/25d3c3b6-2b4a-42c3-8667-60c5f5bf870b)

First version of this just went live for testing on www.kokoanalytics.com :tada:

**Comment by dannyvankooten** [comment]
This was added to [Koko Analytics Pro](https://www.kokoanalytics.com/pricing/) a couple of months ago.
